.github/workflows/main.yml

```diff
@@ -19,19 +19,6 @@ jobs:
         WITH_V: true
         RELEASE_BRANCHES: main
       id: bump
-    - name: Create Release
-      id: create_release
```

Comment:
Why is this block being removed ? Isn't this how we did github actions defined by tiffany . So everytime a docker is updated and pushed a release for the docker is created and pushed to dockerhub ?

Reply:
When we push a new commit to main branch, it will create a new release for us. We may end up having dozens of releases (v0.x.0) if we keep committing many times. We will create a release (v1.0.0) anyway. So I thought we don't need this block.

Reply:
Oh it only creates a new release when there is a new change in the docker file in that commit . Else nothing happens. Like now we have only one release which is when we added the dockerfile 8 hours ago .

Reply:
As per my understanding docker release and your code releases are different .

Reply:
Yes, they are under different steps but using the same tag.
Github release is under actions/create-release@v1
Docker release is under elgohr/Publish-Docker-Github-Action@master

But you are right. It only creates a new release when dockerfile is changed. Let me revert the change

Reply:
Cool 👍 once you revert the change you can merge this . Thanks harry.